x-pack/plugin/src/test/resources/rest-api-spec/test/data_stream/10_data_stream_resolvability.yml

```diff
@@ -32,6 +32,11 @@
         index: logs-foobar
   - match: { error.reason: "No searchable snapshots indices found"}
 
+  - do:
```

Comment:
I think we need a test demonstrating that the explain on a data-stream with indices controlled by ILM return the right output?